sdk/eventhub/azure-eventhub-checkpointstoretable/setup.py

```diff
@@ -51,6 +51,7 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.10',
```

Comment:
3.11?

Reply:
I think checkpointstoretable takes a dependency on eventhub, so you'll prob have to add 3.11 to azure-eventhub too if it's not already there.